docs/aws/README.md

```diff
@@ -11,36 +13,61 @@ export AWS_DEFAULT_OUTPUT=json
 ```
 
 ### Create a Cluster
-Note: If you already have a cluster with version 1.19 or above, you may need to manually tag your subnets for Karpenter to work [detailed here](https://github.com/awslabs/karpenter/issues/404#issuecomment-845283904).
 
-If your cluster version is 1.18 or below, you can skip this step.
+Create an EKS cluster
 ```bash
 eksctl create cluster \
 --name ${CLUSTER_NAME} \
---version 1.18 \
---region ${AWS_DEFAULT_REGION} \
 --node-type m5.large \
 --nodes 1 \
 --nodes-min 1 \
 --nodes-max 10 \
---managed
+--managed \
+--with-oidc
 ```
 
-### Setup IRSA, Karpenter Controller Role, and Karpenter Node Role
-We recommend using [CloudFormation](https://aws.amazon.com/cloudformation/) and [IAM Roles for Service Accounts](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html) (IRSA) to manage these permissions. For production use, please review and restrict these permissions for your use case.
+Tag the cluster subnets with the required tags for Karpenter auto discovery.
+
+Note: If you already have a cluster with version 1.19 or below you can skip this step.
+More [detailed here](https://github.com/awslabs/karpenter/issues/404#issuecomment-845283904).
+
+```bash
+export SUBNET_IDS=$(aws cloudformation describe-stacks \
+    --stack-name eksctl-${CLUSTER_NAME}-cluster \
+    --query 'Stacks[].Outputs[?OutputKey==`SubnetsPrivate`].OutputValue' \
+    --output text)
+
+aws ec2 create-tags \
+    --resources $(echo $SUBNET_IDS | tr ',' '\n') \
+    --tags Key="kubernetes.io/cluster/${CLUSTER_NAME}",Value=
+```
+
+For existing clusters you may need to add an OIDC provider for the cluster.
+We already added one with the `eksctl create cluster` command but it is safe to run again.
```

Comment:
Is there any reason we'd need to be safe? I'm wary of unnecessary steps/complexity. The only case I can imagine here is if a customer brought their own cluster. In that case, we could just add a note by create cluster that irsa is a dependency.

Reply:
I wanted to add this step just in case, but I'm fine changing it to a note instead of a command.